ncm-metaconfig/src/main/metaconfig/slurm/tests/profiles/config.pan

```diff
@@ -107,7 +107,7 @@ prefix "/software/components/metaconfig/services/{/etc/slurm/slurm.conf}/content
 #SchedulerTimeSlice = 30;
 "SchedulerType" = "backfill";
 "SchedulerParameters" = dict(
-    "default_queue_depth", 128,
+    "default_5fqueue_5fdepth", 128,
```

Comment:
Should be:
"{default_queue_depth}", 128,
as per the comment above about how users are expected to specify config?